mod/execution/pkg/deposit/sync.go

```diff
@@ -93,7 +91,7 @@ func (s *Service[
 	deposits, err := s.dc.ReadDeposits(ctx, blockNum)
 	if err != nil {
 		s.metrics.markFailedToGetBlockLogs(blockNum)
-		s.failedBlocks[blockNum] = struct{}{}
+		s.failedBlocks.Store(blockNum, true)
```

Comment:
just store it as an empty struct because we're essentially using the map as a set. its a dumb quirk of the go language:

https://stackoverflow.com/a/57620378

Reply:
Thanks @ocnc